target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -33,6 +33,13 @@ etactica,eg200)
 	ucidef_set_led_oneshot "modbus" "Modbus" "$boardname:red:modbus" "100" "33"
 	ucidef_set_led_default "etactica" "etactica" "$boardname:red:etactica" "ignore"
 	;;
+engenius,ews511ap)
+	ucidef_set_led_default "power" "POWER" "$board:amber:power" "1"
```

Comment:
Can you check if this is indeed $board and not $boardname? 

Because $board would be the first compatible string from the dts (engenius,ews511ap) whereas your LED has the short name: ews511ap:amber:power.

Reply:
It can be dropped at all, since the diag.sh script takes care of the LED due to the dts aliases.

Reply:
Done. Thanks for your reminder.